tgui/packages/tgui/interfaces/NtosNotepad.tsx

```diff
@@ -477,12 +483,18 @@ export const NtosNotepad = (props) => {
         </Box>
       </NtosWindow.Content>
       {activeDialog === Dialogs.UNSAVED_CHANGES && (
-        <UnsavedChangesDialog
```

Comment:
not on tgui core and kind of excessive, i chose to just copy paste what it was here